<launch>

  <arg name="TARGET" default="2" />
  <arg name="PROCESS" default="true" />
  <arg name="USE_PASSTHROUGH" default="true" />

  <node name="static_transform_publisher0"
        pkg="tf" type="static_transform_publisher"
        args="0 0 0 0 0 0 map panda_link0 100">
  </node>

  <include file="$(find morefusion_panda_ycb_video)/launch/setup_static.xml">
    <arg name="PROCESS" value="$(arg PROCESS)" />
    <arg name="USE_PASSTHROUGH" value="$(arg USE_PASSTHROUGH)" />
  </include>

  <group ns="camera">
    <node name="select_picking_order"
          pkg="morefusion_panda_ycb_video" type="select_picking_order.py"
          clear_params="true"
          output="screen" if="$(arg PROCESS)">
      <remap from="~input/camera_info" to="color/camera_info" />
      <remap from="~input/poses" to="with_occupancy/collision_based_pose_refinement/object_mapping/output/poses" />
      <rosparam subst_value="true">
        target: $(arg TARGET)
      </rosparam>
    </node>
  </group>

  <group ns="robot_demo">
    <node name="object_poses_to_markers"
          pkg="morefusion_panda_ycb_video" type="object_poses_to_markers.py"
          clear_params="true">
      <remap from="~input" to="output/placed" />
    </node>
  </group>

</launch>
